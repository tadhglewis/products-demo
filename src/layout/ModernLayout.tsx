import { ReactNode } from 'react';
import styled from 'styled-components';

const Box = styled.div`
  max-width: 920px;
  margin-left: auto;
  margin-right: auto;
  padding: 16px;
`;

const ModernLayout = ({ children }: { children: ReactNode }) => (
  <Box>{children}</Box>
);

export default ModernLayout;
