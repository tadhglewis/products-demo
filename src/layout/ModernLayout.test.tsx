import { render, screen } from '@testing-library/react';
import ModernLayout from './modernLayout';

describe('modernLayout', () => {
  it('should match snapshot', async () => {
    const { container, getByText } = render(
      <ModernLayout>
        <p>Hello</p>
      </ModernLayout>,
    );

    expect(getByText('Hello')).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
