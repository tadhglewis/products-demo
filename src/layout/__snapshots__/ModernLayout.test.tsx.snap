// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modernLayout should match snapshot 1`] = `
.c0 {
  max-width: 920px;
  margin-left: auto;
  margin-right: auto;
  padding: 16px;
}

<div>
  <div
    class="c0"
  >
    <p>
      Hello
    </p>
  </div>
</div>
`;
